lib/components/FormInput.js

```diff
@@ -9,25 +9,29 @@ const FormInput = ({
   placeholder,
   onChange,
   className,
-  value,
+  value = '',
   error,
   children,
   label,
   ...props
 }) => {
-  const [inputValue, setInputValue] = useState('');
+  let timer = null;
+  const [inputValue, setInputValue] = useState(value);
```

Comment:
Why you need to store value as a state in that component ? Especially when it comes from properties - there is two source of truth here.

Can't we remove local state and base only on `value` from props ?

Reply:
performance issue in dashboard builder. If we rely on value from props we need to wait till all changes are applied to the dashboard and then update the input. 

I map value from prop to state on component mount event and treat the state as a single source of truth.